tests/lib/tools/tests.invariant

```diff
@@ -147,7 +161,7 @@ check_invariant() {
 }
 
 main() {
-	ALL_INVARIANTS="root-files-in-home crashed-snap-confine lxcfs-mounted stray-dbus-daemon leftover-defer-sh"
+	ALL_INVARIANTS="root-files-in-home crashed-snap-confine lxcfs-mounted stray-dbus-daemon leftover-defer-sh leftover_user_units"
```

Comment:
is this the order that the invariants get run in? if so we want to run this invariant _before_ the crashed-snap-confine invariant otherwise we won't see the actual error here

Reply:
thanks for the review, yes this is the order but all the invariants are checked and then all the results are displayed in case some invariant check fails. So, if crashed-snap-confine we will see that it fails and then if leftover_user_units fails too we will see which user service is running.

Reply:
ah okay thanks for explaining